modules/statics/src/coins/erc20Coins.ts

```diff
@@ -12579,6 +12579,18 @@ export const erc20Coins = [
     undefined,
     Networks.test.hoodi
   ),
+  terc20(
+    'a0660bd4-c508-4704-ac2b-c7ead1f6ad43',
+    'hteth:stgsofiusd',
+    'STGSoFIUSD',
```

Comment:
Name will be SoFi USD or Test SoFi USD whichever standard we are using for other tokens